Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzill41559.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+using Xamarin.Forms;
+using Xamarin.Forms.CustomAttributes;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+using System.Linq;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Issue(IssueTracker.Bugzilla, 41559, "In iOS Entry.keyboard=Keyboard.Numeric does not have a done button")]
+	public class Bugzill41559 : TestContentPage
```

Comment:
Small thing, but missing the `a` on `Bugzilla` on the class & file name. No need to fix unless other changes are identified.